code-env/python/spec/requirements.txt

```diff
@@ -1,13 +1,9 @@
-azure-common==1.1.27
-azure-identity==1.6.0
-azure-mgmt-authorization==1.0.0
-azure-mgmt-compute==21.0.0
-azure-mgmt-containerregistry==8.0.0
-azure-mgmt-containerservice==16.0.0
-azure-mgmt-core==1.2.2
-azure-mgmt-resource==18.0.0
-azure-mgmt-msi==1.0.0
+azure-identity==1.7.1
+azure-mgmt-authorization==2.0.0
+azure-mgmt-compute==23.1.0
+azure-mgmt-containerservice==16.4.0
+azure-mgmt-resource==20.0.0
+azure-mgmt-msi==6.0.0b1
 ipaddress==1.0.23
-msrest==0.6.21
```

Comment:
Some info about this removal since it was explicitly added in sc-93743
Some exception where moved from msrest into azure-mgmt-core. And we run into that issue because we where pining azure-mgmt-core.

https://github.com/Azure/azure-sdk-for-python/issues/24765#issuecomment-1152709594